aws-enigma-2/main.tf

```diff
@@ -0,0 +1,205 @@
+variable "aws_heroku_org" {}
+
+variable "env" {
+  default = "enigma"
```

Comment:
I'm pretty sure this should be `"production"`.  Did you run into issues with it being that instead of `"enigma"`?